scripts/testing/xbox-ssl.conf

```diff
@@ -104,42 +89,22 @@
 
     # Trailing slash required.
     RewriteRule ^/dxos/wns$ /dxos/wns/ [R]
-    <Location /dxos/wns/>
-      SetEnv proxy-sendcl
-      ProxyPass "http://localhost:9473/"
-      ProxyPassReverse "http://localhost:9473/"
-      Allow from all
-    </Location>
+    RewriteRule ^/dxos/wns/(.*) http://localhost:9473/$1 [P,L]
 
     #
     # IPFS
     #
 
     RewriteRule ^/dxos/ipfs/api$ /dxos/ipfs/api/ [R]
-    RewriteRule /dxos/ipfs/api/v([0-9]+)/(.*)           http://localhost:5001/api/v$1/$2 [P,L]
-    <Location /dxos/ipfs/api/>
-      SetEnv proxy-sendcl
-      ProxyPass "http://localhost:5001/api/v0/"
-      ProxyPassReverse "http://localhost:5001/api/v0/"
-      Allow from all
-    </Location>
+    RewriteRule ^/dxos/ipfs/api/v([0-9]+)/(.*) http://localhost:5001/api/v$1/$2 [P,L]
```

Comment:
Add a one-line comment for each non-obvious rule. e.g. I'm guessing the funky 0-9 regex is needed for some reason. What is that reason?